azure-keyvault-complete/pom.xml

```diff
@@ -7,13 +7,13 @@ the MIT License. See License.txt in the project root for license information. --
   <parent>
 		<groupId>com.microsoft.azure</groupId>
 		<artifactId>azure-keyvault-parent</artifactId>
-		<version>1.1.2</version>
+		<version>1.1.3</version>
```

Comment:
If there's a functionally breaking change, should we at least do a minor version release, not a patch release?